src/libaktualizr/uptane/uptane_network_test.cc

```diff
@@ -139,6 +140,44 @@ TEST(UptaneNetwork, DownloadFailure) {
   EXPECT_TRUE(result.first);
 }
 
+/*
+ * Output a log when connectivity is restored.
+ */
+TEST(UptaneNetwork, LogConnectivityRestored) {
+  //  testing::internal::CaptureStdout();
+  RecordProperty("zephyr_key", "OTA-991,TST-158");
```

Comment:
We can leave out this `RecordProperty` thing for now. Maybe someday that'll be important again, but for now, it isn't.